From 6a4821ce02cb2f1e54cdec9c2e121fb5ca0ddd96 Mon Sep 17 00:00:00 2001
From: Marsell Kukuljevic <marsell@joyent.com>
Date: Tue, 10 Jul 2018 17:30:34 +0200
Subject: [PATCH] TRITON-556 Update coal-setup.sh Reviewed by: Pedro P. Candel
 <pedro@joyent.com> Reviewed by: Trent Mick <trentm@gmail.com> Approved by:
 Trent Mick <trentm@gmail.com>

---
 tools/coal-setup.sh | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/tools/coal-setup.sh b/tools/coal-setup.sh
index 2e12da5..b063eb0 100755
--- a/tools/coal-setup.sh
+++ b/tools/coal-setup.sh
@@ -134,6 +134,9 @@ hack_imgapi_to_allow_local_custom_images
 echo "# Create test_128 package"
 create_test_128_package
 
+echo "# Import dev-sample-data, used by some tests"
+sdcadm post-setup dev-sample-data
+
 # Current the cloudapi test suite assumes the following image is installed:
 #   minimal-64-lts (latest version)
 # Note: Keep in sync with test/common.js#getBaseImage().
@@ -185,7 +188,12 @@ fi
 echo "# Enable volapi"
 sdcadm post-setup volapi
 sdcadm experimental nfs-volumes cloudapi
+sdcadm experimental nfs-volumes cloudapi-automount
 sdcadm experimental nfs-volumes docker
+sdcadm experimental nfs-volumes docker-automount
+
+echo "# Enable cns"
+sdcadm post-setup cns
 
 if ! $(nictagadm exists sdc_underlay 2>/dev/null); then
     echo "# Enable external and underlay nics"
-- 
2.21.0

